db/migrate/20240626222401_create_state_file_wa_intakes.rb

```diff
@@ -0,0 +1,53 @@
+class CreateStateFileWaIntakes < ActiveRecord::Migration[7.1]
+  def change
+    create_table :state_file_wa_intakes do |t|
+      # misc
+      t.integer :tax_return_year
+      t.integer :filing_status
```

Comment:
this is wrong!! the intake should not have a `filing_status` column. it only looks like it does in places because the base intake class implements [a method that pulls it from the direct file data](https://github.com/codeforamerica/vita-min/blob/main/app/models/state_file_base_intake.rb#L97-L105)